test/rules/csslint.spec.js

```diff
@@ -1,28 +0,0 @@
-const expect = require('expect.js');
-
-const HTMLHint = require('../../dist/htmlhint.js').default;
-
-const ruldId = 'csslint';
-const ruleOptions = {};
-
-ruleOptions[ruldId] = {
-  'display-property-grouping': true,
-  'known-properties': true
-};
-
-describe(`Rules: ${ruldId}`, function() {
-  it('should result in an error', function() {
```

Comment:
If we want a replacement later, we could use `skip` instead of removing this file